src/core/ext/transport/cronet/transport/cronet_transport.cc

```diff
@@ -555,6 +555,19 @@ static void on_response_headers_received(
   for (size_t i = 0; i < headers->count; i++) {
     if (0 == strcmp("grpc-status", headers->headers[i].key)) {
       on_response_trailers_received(stream, headers);
+      /* Do an extra read for a trailer-only stream with grpc_status = 0
+       to trigger on_succeeded() callback */
+      if (0 == strcmp(headers->headers[i].value, "0")) {
+        stream_obj* s = static_cast<stream_obj*>(stream->annotation);
```

Comment:
This is unnecessary since you already have `s` from line 551.

Reply:
Good catch, done.